.travis.yml

```diff
@@ -50,10 +56,13 @@ script:
     - git remote add binaries ${BINARIES_REPO}
     - git fetch binaries
     - git checkout -b travis-builds binaries/travis-builds
+    - git lfs track skydive-latest
```

Comment:
travis is not used anymore, don't know where you saw the error ?

Reply:
it's on my personal account, so lfs is not enabled as I have a free account
I was used travis, as it's sometime faster than wait for our CI
.... I think we can remove .travis.yml file